pkg/collector/collector.go

```diff
@@ -733,3 +766,58 @@ func openEventExists(openEvent *tracing.OpenEvent, openEvents []OpenCalls) (bool
 
 	return hasSamePath, hasSameFlags
 }
+
+func (cm *CollectorManager) shouldIncludeOpenEvent(openEvent *tracing.OpenEvent, openEvents []OpenCalls) bool {
+	// Check if we exceeded the maximum number of open events.
+	if len(openEvents) > MaxOpenEvents {
+		return false
+	}
+
+	// Check if we should ignore this path.
+	if os.Getenv("OPEN_IGNORE_PREFIXES") != "" {
```

Comment:
The best practice to do these configs in main, not in a nested package

Reply:
Done